Makefile

```diff
@@ -1,9 +1,50 @@
 all: test
 
+# Packages to install if we are running on Debian
+PACKAGES := \
+    python3-django \
+    python3-flake8 \
+    python3-jinja2 \
+    python3-pytest-django \
+    python3-gunicorn \
+    python3-coverage \
+    python3-pytest-cov \
+    python3-django-redis \
+    python3-ipy \
+    python3-django-ratelimit \
+    python3-django-cors-headers \
+    python3-django-oauth-toolkit \
+    redis-server \
+    gunicorn3 \
+    python3-gevent \
+
+# TODO - from the requirements.txt
+# pyserial>=3.2.1
+# RPi.GPIO>=0.6.3
```

Comment:
This one is called `rpi.gpio` on `raspbian`.
It's only needed if your runtime env is a raspberry pi.

For testing there is a mock implementation: https://github.com/dimsumlabs/hackman/blob/8019b23736518573efd0444dd2a6a8ac3c00c8f6/hackman_door/door/impls/dummy_test.py